### PR TITLE
1.3 eval filter

### PR DESCRIPTION
**Pull request by BryonLewis**
Taking #338 and made some changes for the UI and fixing some Warnings/errors

- No Eval for RGD endpoint error
- fixes Performer warnings for recursive lookups
- ran pre-commit and linting
- Added page sizes for region endpoints of 1000 (when loading all annotations in RGD you get more than 100 regions).

Changed some vuetify logic to make the filters look like this: 
image

### CODE REVIEW

File: django/src/rdwatch_scoring/views/eval.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from ninja.pagination import RouterPaginated
+
+from django.db.models import CharField, Value
+from django.db.models.functions import Concat
+from django.http import HttpRequest
+
+from rdwatch_scoring.models import EvaluationRun
+
+router = RouterPaginated()
```
Comment (BryonLewis) [inline django/src/rdwatch_scoring/views/eval.py:9]: Should this be updated for the case where there are more than 100 evaluations?  I did some stuff to default regions to 1000, should this be increased from the stock 100?

**Review by BryonLewis — COMMENTED** [review]
Couple of internal comments on pagination and if we should change the name `/eval/` to something like `/evalnum/` to it isn't confused with evaluationRuns.

File: django/src/rdwatch_scoring/views/eval.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    return (
+        EvaluationRun.objects.annotate(
+            eval=Concat(
+                'evaluation_number',
+                Value('.'),
+                'evaluation_run_number',
+                output_field=CharField(),
+            )
+        )
+        .order_by('evaluation_number', 'evaluation_run_number')
```
Comment (BryonLewis) [inline django/src/rdwatch_scoring/views/eval.py:23]: I think we may actually want these in descending order.  So the newest is at the top?

File: django/src/rdwatch_scoring/api.py [hunk 3]
```diff
@@ -11,6 +11,7 @@
 api.add_router('/performers/', views.performer.router)
 api.add_router('/regions/', views.region.router)
 api.add_router('/sites/', views.site.router)
+api.add_router('/evals/', views.eval.router)
```
[the remark this answers is not in the document]
Comment (mvandenburgh) [reply django/src/rdwatch_scoring/api.py:14]: I agree, `/evalnums` or `/eval-nums` would be clearer

File: django/src/rdwatch_scoring/views/eval.py (hunk 1 above)
Comment (mvandenburgh) [reply django/src/rdwatch_scoring/views/eval.py:9]: I think that would be fine. 100 vs 1000 wouldn't make a huge difference i would think

File: django/src/rdwatch_scoring/views/eval.py (hunk 2 above)
Comment (mvandenburgh) [reply django/src/rdwatch_scoring/views/eval.py:23]: Agreed.

File: django/src/rdwatch_scoring/views/eval.py (hunk 1 above)
Comment (mvandenburgh) [reply django/src/rdwatch_scoring/views/eval.py:9]: Oh, i see this is already done

**Comment by mvandenburgh** [comment]
Updated this to 
1) Change `/evals` endpoint to `/eval-numbers`	for clarity
2) Order evaluation numbers in descending order, so the newest one comes first.

**Review by mvandenburgh — APPROVED** [review]

**Pull request merged**